qdrant-landing/content/documentation/cloud/aws-backups.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Back up your AWS cluster
+weight: 65
+---
+
+## Back up your AWS cluster
+
+You can back up your Qdrant clusters deployed on AWS though the Qdrant Cloud
+Dashboard at https://cloud.qdrant.io. This section assumes that you've already
+set up your cluster, as described in the following sections:
+
+- [Create a cluster](/documentation/cloud/create-cluster/)
+- Set up [Authentication](/documentation/cloud/authentication/)
+- Configure one or more [Collections](/documentation/concepts/collections/)
+
+## Configure a backup
+
+After you have taken the prerequisite steps, you can configure a backup with the
+[Qdrant Cloud Dashboard](https://cloud.qdrant.io). To do so, take these steps:
+
+1. Sign in to the dashboard
+1. Select Clusters.
+1. Select the cluster that you want to back up.
+   ![Select a cluster](/documentation/cloud/select-cluster.png)
+1. Find and select the **Backups** tab.
+1. Now you can set up a backup schedule.
+   The **Days of Retention** is the number of days before the backup is deleted.
+1. Alternatively, you can select **Backup now**.
+
+![Configure a cluster backup](/documentation/cloud/backup-schedule.png)
+
+## Restore a backup
+
+If you have a backup, it appears in the list of **Available Backups**. You can
+choose to restore or delete the backups of your choice.
+
```

Comment:
Few information I think missing from this doc:
1. The backup is incremental in nature. So, only the modified data since last backup will be uploaded to backup storage. This will reduce user's total cost of backup.
2. You can create multiple backup schedules.
3. When you restore a snapshot, data inserted after taking this snapshot will get lost.

Reply:
I've added a new section with this info to the end of the backups.md file.